service-api/module/Application/src/Yoti/YotiService.php

```diff
@@ -271,9 +271,10 @@ public function letterConfigPayload(CaseData $caseData, string $requirementId):
         ];
         /** @var CounterService $counterServiceData */
         $counterServiceData = $caseData->counterService;
+        $postOfficeData = json_decode($counterServiceData->selectedPostOffice, true);
```

Comment:
I hadn't spotted that we were stored `selectedPostOffice` as a JSON blob. Not a blocker for this PR but [I've put in a ticket](https://opgtransform.atlassian.net/browse/ID-295) to revisit that